packages/zod/src/helpers/util.ts

```diff
@@ -38,11 +40,11 @@ export const getValidEnumValues = (obj: any) => {
     filtered[k] = obj[k];
   }
   return objectValues(filtered);
-};
+}
 
-export const objectValues = (obj: any) => {
+export function objectValues(obj: any): any {
```

Comment:
Just out of pure curiosity, what's the reason for switching to functions here? Is it a requirement of JSR's "fast types" (or isolatedDeclarations)?

Reply:
Nope, just slowly migrating more arrow functions to `function` because I think it's prettier

Reply:
Agree it's prettier🔥🔥🔥. Think at the time i must have just been curious if/how it would affect isolatedDeclarations